notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p><s>Stable Diffusion </s>DeepFoyd IF in Diffusers library</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>😂corrected!</p>